### PR TITLE
point to new doc server and 4.0 docs

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Email send with outdated link to the documentation.
Just update the link to 4.0 docs.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **just point to new docs**

- [x] **DONE**

## Test coverage
- No tests: **no tests for docs**

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by mcalmer** [comment]
I think it could make sense to port this to 4.0 branch.

File: backend/server/rhnServer/server_hardware.py [hunk 1]
```diff
@@ -1231,5 +1231,5 @@ def reload_hardware_byid(self, sysid):
 
 
 For more information on generating a new machine_id for traditional systems refer to:
-https://www.suse.com/documentation/suse-manager-3/book_suma_best_practices_31/data/bp_chapt_suma3_troubleshooting_registering_cloned_traditional_systems.html
+https://documentation.suse.com/external-tree/en-us/suma/4.0/suse-manager/administration/tshoot-registerclones.html
```
Comment (juliogonzalez) [inline backend/server/rhnServer/server_hardware.py]: Mmmmmm, shouldn't this point to:
https://www.uyuni-project.org/uyuni-docs/suse-manager/suse-manager/administration/tshoot-registerclones.html if the server is Uyuni?

File: java/code/src/com/suse/manager/webui/menu/MenuTree.java [hunk 2]
```diff
@@ -395,7 +395,7 @@
                     .withPrimaryUrl("https://www.suse.com/support/kb/product.php?id=SUSE_Manager")
                     .withTarget("_blank"))
                 .addChild(new MenuItem("header.jsp.documentation")
-                    .withPrimaryUrl("https://www.suse.com/documentation/suse_manager/")
+                    .withPrimaryUrl("https://documentation.suse.com/suma/")
```
Comment (juliogonzalez) [inline java/code/src/com/suse/manager/webui/menu/MenuTree.java]: Same question.

File: java/code/src/com/suse/manager/webui/menu/MenuTree.java [hunk 3]
```diff
@@ -435,7 +435,7 @@
...
```
Comment (juliogonzalez) [inline java/code/src/com/suse/manager/webui/menu/MenuTree.java]: And same.

**Comment by mcalmer** [comment]
makes sense. I have changed the PR to differentiate between SUMA and Uyuni.

**Pull request merged**